addon-test-support/@ember/test-helpers/settled.ts

```diff
@@ -149,6 +150,7 @@ export interface SettledState {
   hasPendingRequests: boolean;
   hasPendingTransitions: boolean | null;
   pendingRequestCount: number;
+  debugInfo?: DebugInfo;
```

Comment:
I’d like this to reference `IDebugInfo` instead of the class itself...

Reply:
Done.